Doc/library/datetime.rst

```diff
@@ -606,8 +583,13 @@ Supported operations:
 +-------------------------------+----------------------------------------------+
 | ``timedelta = date1 - date2`` | \(3)                                         |
 +-------------------------------+----------------------------------------------+
-| ``date1 < date2``             | *date1* is considered less than *date2* when |
-|                               | *date1* precedes *date2* in time. (4)        |
+| ``date1 == date2``            | Equality comparison. (4)                     |
+| ``date1 != date2``            |                                              |
++-------------------------------+----------------------------------------------+
+| ``date1 < date2``             | Order comparison. (5)                        |
+| ``date1 > date2``             |                                              |
+| ``date1 <= date2``            |                                              |
+| ``date1 >= date2``            |                                              |
```

Comment:
This ^ renders like:

<img width="816" alt="image" src="https://github.com/python/cpython/assets/1324225/3818c306-e62b-492f-947c-4e3dbfd91524">

This:


```suggestion
| ``date1 == date2``            | Equality comparison. (4)                     |
|                               |                                              |
| ``date1 != date2``            |                                              |
+-------------------------------+----------------------------------------------+
| ``date1 < date2``             | Order comparison. (5)                        |
|                               |                                              |
| ``date1 > date2``             |                                              |
|                               |                                              |
| ``date1 <= date2``            |                                              |
|                               |                                              |
| ``date1 >= date2``            |                                              |
```

Renders like:

<img width="774" alt="image" src="https://github.com/python/cpython/assets/1324225/f78e1e7b-dbbf-4fb6-b417-66e59fbe0e2c">

And this:

```suggestion
| ``date1 == date2``            | Equality comparison. (4)                     |
+-------------------------------+                                              +
| ``date1 != date2``            |                                              |
+-------------------------------+----------------------------------------------+
| ``date1 < date2``             | Order comparison. (5)                        |
+-------------------------------+                                              +
| ``date1 > date2``             |                                              |
+-------------------------------+                                              +
| ``date1 <= date2``            |                                              |
+-------------------------------+                                              +
| ``date1 >= date2``            |                                              |
```

Renders like:

<img width="774" alt="image" src="https://github.com/python/cpython/assets/1324225/9df7ae57-1a0b-4f6c-a6bc-cb1c1643fc3d">

Same applies to the `datetime` table below.

Reply:
Good catch!

I fixed this using other method.